package.json

```diff
@@ -17,6 +17,7 @@
     "escape-goat": "4.0.0",
     "fast-glob": "3.2.11",
     "font-awesome": "4.7.0",
+    "is-typedarray": "1.0.0",
```

Comment:
I've added that because there was a failure on "lint-frontend"

![image](https://user-images.githubusercontent.com/98962410/171602404-df0cb316-4272-4a89-a261-daab9f2c9e14.png)